src/OrleansManager/Microsoft.Orleans.OrleansManager.targets

```diff
@@ -0,0 +1,21 @@
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Target Name="CopyOrleansManagerTools" BeforeTargets="AfterBuild">
+    <ItemGroup>
+      <FilesToCopy Include="$(MSBuildThisFileDirectory)..\tools\OrleansManager.exe.config" />
+      <FilesToCopy Include="$(MSBuildThisFileDirectory)..\tools\ClientConfiguration.xml" />
```

Comment:
I don't know. We only supply this in a Nuget for users who want no hassle usage, like OrleansHost.exe. The Nuget is designed to run from Output directory, but Nuget was not copying that file.
With the current setup the user can overwrite this file in an AfterBuild or later target if needed, but I don't think that anyone runs it from output directory in a live environment ;-)